packages/cells/src/widget.ts

```diff
@@ -1762,6 +1770,7 @@ export class MarkdownCell extends AttachmentsCell<IMarkdownCellModel> {
   private _rendered = true;
   private _prevText = '';
   private _ready = new PromiseDelegate<void>();
+  private _showEditorForReadOnlyMarkdown: boolean = true;
```

Comment:
This is strange that it was not flagged by the linter. But the rule is to not specify type for variable with default value and basic type (see e.g. `_rendered` or `_prevText` above):

```suggestion
  private _showEditorForReadOnlyMarkdown = true;
```